internal/storage/v1/elasticsearch/factory_test.go

```diff
@@ -414,3 +473,42 @@ func TestPasswordFromFileErrors(t *testing.T) {
 	require.NoError(t, os.Remove(pwdFile))
 	f.onPasswordChange()
 }
+
+func TestFactoryBase_NewClient_WatcherError(t *testing.T) {
+	testCases := []struct {
```

Comment:
please simplify. You are doing a single test, no need to have it from a data table or have the complexity of setup/cleanup functions.